lib/streamlit/elements/vega_charts.py

```diff
@@ -920,6 +929,18 @@ def area_chart(
 
         """
 
+        # Check that the stack parameter is valid, raise more informative error message if not
+        maybe_raise_stack_warning(
+            stack,
+            "st.area_chart",
+            "https://docs.streamlit.io/develop/api-reference/charts/st.area_chart",
+        )
+
+        # st.area_chart's stack=False option translates to a "layered" area chart for vega. We reserve stack=False for
+        # grouped/non-stacked bar charts, so we need to translate False to "layered" here.
+        if stack is False:
+            stack = "layered"
```

Comment:
If someone changes the value here by accident, how would we catch it? If we don't, we should probably have unit tests ensuring this behavior.